File: backend/src/main/java/com/lawencon/ticketms/dto/ticketpriority/TicketPriorityDataDto.java
package com.lawencon.ticketms.dto.ticketpriority;

public class TicketPriorityDataDto {
	private Long id;
	private String priorityCode;
	private String priority;
	private Integer ver;
	public Long getId() {
		return id;
	}
	public void setId(Long id) {
		this.id = id;
	}
	public String getPriorityCode() {
		return priorityCode;
	}
	public void setPriorityCode(String priorityCode) {
		this.priorityCode = priorityCode;
	}
	public String getPriority() {
		return priority;
	}
	public void setPriority(String priority) {
		this.priority = priority;
	}
	public Integer getVer() {
		return ver;
	}
	public void setVer(Integer ver) {
		this.ver = ver;
	}
}

File: frontend/src/app/dto/ticket-priority/ticket-priority.data.dto.ts
export class TicketPriorityDataDto{
    private _id! : number
	private _priorityCode! : string
	private _priority! : string
	private _ver! : number

    public get id(): number {
        return this._id;
    }

    public set id(id: number) {
        this._id = id;
    }

    public get priorityCode(): string {
        return this._priorityCode;
    }

    public set priorityCode(priorityCode: string) {
        this._priorityCode = priorityCode;
    }

    public get priority(): string {
        return this._priority;
    }

    public set priority(priority: string) {
        this._priority = priority;
    }

    public get ver(): number {
        return this._ver;
    }

    public set ver(ver: number) {
        this._ver = ver;
    }

}